### PR TITLE
Implement anonymous connection

### PR DESCRIPTION
**Pull request by StevenWatremez**

### CODE REVIEW

File: ZetaPushSwift/ClientHelper.swift [hunk 1]
```diff
@@ -88,56 +88,59 @@ open class ClientHelper : NSObject, CometdClientDelegate{
...
+          
+          print("ZP server -> target url : " + url!.description)
             
             let task = URLSession.shared.dataTask(with: url!) { data, response, error in
-                
+            
+              print("ZP server -> server response data : " + data!.description)
+              let jsonAnyTest = try? JSONSerialization.jsonObject(with: data!, options: [])
+              let jsonTest = jsonAnyTest as! [String : AnyObject]
+              print("ZP server -> server response : " + jsonTest.description)
```
Comment (anthonyGuiguen) [inline ZetaPushSwift/ClientHelper.swift]: You can use `self.log`

File: ZetaPushSwift/WebsocketTransport.swift [hunk 2]
```diff
@@ -11,83 +11,130 @@ import Foundation
...
+    self.init()
+    self.urlString = url
+    log.setup(level: logLevel)
+  }
+  
+  func openConnection() {
+    self.closeConnection()
+    self.webSocket = WebSocket(url: URL(string:self.urlString!)!)
+    if let webSocket = self.webSocket {
+//      webSocket.delegate = self
```
Comment (anthonyGuiguen) [inline ZetaPushSwift/WebsocketTransport.swift]: Why it's commented ?

File: ZetaPushSwift/WebsocketTransport.swift [hunk 3]
```diff
@@ -11,83 +11,130 @@ import Foundation
...
+//    self.delegate?.didReceiveMessage(text)
+//  }
+//
+//  // MARK: TODO
+//  internal func websocketDidReceiveData(socket: WebSocketClient, data: Data) {
+//    log.debug("Cometd: Received data: \(data.count)")
+//    //self.socket.writeData(data)
+//  }
+//
+//}
```
Comment (anthonyGuiguen) [inline ZetaPushSwift/WebsocketTransport.swift]: You can remove these lines if you don't use this code

File: ZetaPushSwift/WebsocketTransport.swift [hunk 4]
```diff
@@ -11,83 +11,130 @@ import Foundation
...
+  }
+  
+  func websocketDidReceiveMessage(socket: WebSocket, text: String, response: WebSocket.WSResponse) {
+    log.debug("Advanced transport delegate: message: \(text), from response : \(response)")
+    self.delegate?.didReceiveMessage(text)
+  }
+  
+  func websocketDidReceiveData(socket: WebSocket, data: Data, response: WebSocket.WSResponse) {
+    log.debug("Advanced transport delegate: Received data: \(data.count), from response : \(response)")
+    //self.socket.writeData(data)
```
Comment (anthonyGuiguen) [inline ZetaPushSwift/WebsocketTransport.swift]: Why it's commented ?

**Review by anthonyGuiguen — APPROVED** [review]

**Review by jgoas — APPROVED** [review]

**Pull request closed without merging**